assets/src/js/Components/ArchivePicker/blockUpdateFunctions.js

```diff
@@ -39,3 +39,59 @@ export const updateHappyPointAttributes = id => {
   // Need to update only the id property for the Happy Point block
   return {attributes: {id}};
 };
+
+export const updateMediaAndTextAttributes = (image, currentBlock) => {
+  const div = document.createElement('div');
+
+  if (currentBlock.originalContent) {
+    div.innerHTML = currentBlock.originalContent;
+    div.querySelector('img').setAttribute('class', `wp-image-${image.id}`);
+    div.querySelector('img').src = image.source_url;
+  } else {
+    div.innerHTML = `
+    <div class="wp-block-media-text is-stacked-on-mobile">
+      <figure class="wp-block-media-text__media">
+        <img src=${image.source_url} alt=${image.alt_text} class="wp-image-${image.id} size-full"/>
+      </figure>
+      <div class="wp-block-media-text__content">
```

Comment:
Is it necessary this empty `div`?

Reply:
In my testing it was necessary to make sure we are passing the element properly. I use it to make sure the HTML is rendered properly and we can change specific attributes of the HTML element.